src/openag_brain/software_modules/video_writer.py

```diff
@@ -4,21 +4,72 @@
 the recipe_start data point
 """
 import os
-import time
 import rospy
 import tempfile
 import subprocess
 from openag.cli.config import config as cli_config
 from openag.couch import Server
 from openag.db_names import ENVIRONMENTAL_DATA_POINT
 from openag.var_types import RECIPE_START, RECIPE_END, AERIAL_IMAGE
+from openag_brain.video_helpers import *
+
+IMAGE_ATTACHMENT = "image"
+TIMELAPSE_ATTACHMENT = "timelapse"
+
+IMAGE_FILENAME = "image.png"
+WORKING_TIMELAPSE_FILENAME = "timelapse.mpeg"
+TIMELAPSE_FILENAME = "timelapse.mp4"
+SNIPPET_FILENAME = "snippet.mpeg"
+
+# By default, each second of the timelapse will correspond to 1 day in real
+# time
+DEFAULT_TIMELAPSE_SCALING_FACTOR = 24*60*60
```

Comment:
Nit: the name of this variable made me think it was for dimensional scaling (x,y) of video file.